packages/builder/src/helpers/planTitle.js

```diff
@@ -25,3 +25,9 @@ export function getFormattedPlanName(userPlanType) {
   }
   return `${planName} Plan`
 }
+
+export function emailSupportCheck(userPlanType) {
```

Comment:
This check isn't necessarily restricted to email support, so you could rename the function to `isPremiumAndAbove`, and then have a one line return statement in the function body.

Reply:
Also just wondering if Pro and Team plan types are given premium support or not?

Reply:
PRO and Teams I believe are older plans and didn't have it then. I've asked sales and they agree that they shouldn't have access to it until they move to the new premium plan.